extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -64,5 +88,31 @@ export function adsenseIsA4AEnabled(win, element, useRemoteHtml) {
     addExperimentIdToElement(experimentId, element);
     forceExperimentBranch(win, ADSENSE_A4A_EXPERIMENT_NAME, experimentId);
   }
+
+  // If not in the unconditioned canonical experiment, attempt to
```

Comment:
This function assumes that randomlySelectUnconditionedExperiments ran before it which seems brittle.  Consider combining these.